modules/standard/AutoMath.chpl

```diff
@@ -688,32 +688,78 @@ module AutoMath {
     return m / n;
   }
 
+  // When removing this deprecated function, be sure to remove chpl_erf and
+  // move its contents into Math.chpl to reduce the symbols living in this
+  // module.
+  pragma "last resort"
+  @chpldoc.nodoc
```

Comment:
Should these be documented so that the deprecation warning is also visible in the docs?

Reply:
Oops, yes, you're right!